### ISSUE TITLE
plugin errors when parsing "empty" types

### ISSUE THREAD

**Issue by kevinbarabash** [brigand/babel-plugin-flow-react-proptypes#213]
Specifically the error I get is:
```
Error: babel-plugin-flow-react-proptypes: Encountered an unknown node in the type definition. Node: {"type":"EmptyTypeAnnotation","start":541,"end":546,"loc":{"start":{"line":22,"column":11},"end":{"line":22,"column":16}}}
```
I don't think prop-types has a type that makes sense so I think just ignoring this type makes sense.

**Comment by brigand** [comment]
Thanks for the report. For reference, example code that caused the issue:

```js
type Props = { foo: empty }
```

Fixed in 25.1.0.

**Comment by kevinbarabash** [comment]
Thanks for the quick fix.
